unit-tests/src/org/commcare/android/tests/caselist/EntityListCalloutDataTest.java

```diff
@@ -75,6 +75,7 @@ public void testAttachCalloutResultToListTest() {
 
         // ensure that the entity list is filtered by the received callout
         // result data (fingerprint identification list with confidence score)
+        loadList();
```

Comment:
Got it, like failing in this PR or just in general? Seems fine either way, didn't mean to nitpick, just wasn't sure where it came from/if it was intentional.